xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
```

Comment:
There is a fieldset wrapping radio buttons. Why fieldset here as it doesn't have related elements ?, it just contains the actual form elements.